website/client/src/mixins/groupsUtilities.js

```diff
@@ -75,11 +91,11 @@ export default {
       }
 
       if (filters.guildSize && filters.guildSize.indexOf('gold_tier') !== -1) {
-        correctSize = group.memberCount > 1000;
+        correctSize = group.memberCount >= 1000;
```

Comment:
why the change from > to >=?

Reply:
@paglias Because from the [Habitica Wiki on Guilds Guide](https://habitica.fandom.com/wiki/Guilds_Guide), it says "Large Guilds of Habitica Guilds with 1000 or more members.  I take it this corresponds to the gold tier guilds, and 1000 or more members is equivalent to "membercount >= 1000 instead of > 1000"?

Reply:
makes sense, good catch!